hazelcast/src/main/java/com/hazelcast/security/permission/ActionConstants.java

```diff
@@ -83,9 +83,9 @@ public final class ActionConstants {
     public static final String ACTION_DROP_VIEW = "drop-view";
     public static final String ACTION_CREATE_TYPE = "create-type";
     public static final String ACTION_DROP_TYPE = "drop-type";
-    public static final String ACTION_VIEW_LINK = "view-link";
-    public static final String ACTION_CREATE_LINK = "create-link";
-    public static final String ACTION_DROP_LINK = "drop-link";
+    public static final String ACTION_VIEW_DATACONNECTION = "view-dataconnection";
```

Comment:
dataconnection or data-connection? Idk what fits better, it's two separate words, but describing one thing